dockers/docker-syncd-mlnx/start.sh

```diff
@@ -8,6 +8,10 @@ function clean_up {
 
 trap clean_up SIGTERM SIGKILL
 
+# fw-upgrade will exit if firmware was actually upgraded or if some error
+# occures
+. mlnx-fw-upgrade.sh
```

Comment:
Yes, correct
If firmware is actually updated, mlnx-fw-upgrade.sh will exit, causing syncd service to restart